1-JavaScript/Week3/homework/ex6-totalCost.js

```diff
@@ -20,20 +20,27 @@ instead!
 -----------------------------------------------------------------------------*/
 const cartForParty = {
   // TODO complete this object
+  beer: 2,
+  bread: 3,
+  cheese: 5.3,
+  cola: 2.2,
+  water: 2.39,
 };
 
-function calculateTotalPrice(/* TODO parameter(s) go here */) {
-  // TODO replace this comment with your code
+function calculateTotalPrice(object) {
+  const sumValues = (object) => Object.values(object).reduce((a, b) => a + b);
+  const sum = sumValues(object);
```

Comment:
These two lines can be combined into one (you're now creating a function and calling it directly).
```suggestion
  const sum = Object.values(object).reduce((a, b) => a + b);
```